examples/pod_namespace_watch.py

```diff
@@ -31,6 +31,25 @@ def main():
     v1 = client.CoreV1Api()
     count = 10
     w = watch.Watch()
+    """
+    [INFO: `server-side` & `client-side` timeout]
+
+    `kwargs['timeout_seconds'] = n`:
+       - It is a server-side timeout, i.e. a polite request to the server,
+         mentioning it to cleanly close the connection after n seconds.
+       - If the user have a network outage, then this parameter will do nothing.
```

Comment:
watch is for server to push events to client side, so it is not accurate to say 'user have a network usage'